arthur_bench/scoring/exact_match.py

```diff
@@ -16,24 +17,29 @@ def __init__(self, case_sensitive=True):
     def name() -> str:
         return "exact_match"
 
+    def to_dict(self, warn=False):
```

Comment:
This will override the abstract class implementation, which currently saves some parameters. For example `case_sensitive: True` in this case. We should extend the dictionary instead of overriding.

Additionally, I see you have removed the `CategoricalScorer` subclass. Alternatively to implementing all these `to_dict`methods independently and expecting users to do the same for their implementations, you could bring back the `CategoricalScorer` and provide a default implementation

Reply:
added the CategoricalScorer class back